pkg/collector/corechecks/gpu/gpu.go

```diff
@@ -90,14 +85,14 @@ func (m *Check) Run() error {
 		return err
 	}
 
-	sysprobeData, err := m.sysProbeUtil.GetCheck(sysconfig.GPUMonitoringModule)
+	snd, err := m.GetSender()
```

Comment:
let's keep the defer right after successfully retrieving the sender. for now it is not important, but once nvml collector will be added it will be, so to avoid missing this in the future...

Reply:
Fixed, I thought I had already done it but it seems to be in another PR.